op-batcher/justfile

```diff
@@ -17,7 +17,13 @@ clean:
     rm -f {{BINARY}}
 
 # Run tests
-test: (go_test "./...")
+test: test-failpoint
+    just go_test "./..."
+
+# Failpoint tests rely on https://github.com/pingcap/failpoint, which are identified by a _FailpointTest suffix to the test name.
+# We run them using a separate GOCACHE because they inject code into the compiled binaries, which could interfere with other tests or normal builds.
+test-failpoint:
+    GOCACHE=/tmp/failpoint-cache just go_test "./batcher/..." "-toolexec" "$(mise which failpoint-toolexec)" "-run" ".*_FailpointTest$"
```

Comment:
@litt3 would like to chat tomorrow about this specific test framework needed for a single test. Considering just removing that test, would like to discuss.

Reply:
Conclusion: https://github.com/Layr-Labs/optimism/pull/21#issuecomment-2721688570